### PR TITLE
[Manual Search] Rename variables: forced search and manual search

### PR DESCRIPTION
**Pull request by fernandog**
if need to change name again will be easier as now everything has unique vars names:
forced search
manual search

I guess I covered all vars. Running this branch as my production. 

I wont fix this QC issues:
- Avoid unused arguments
- Avoid using "non-Pythonic" function names

TODO:
- [ ] New manual search icon @medariox

### CODE REVIEW

File: gui/slick/views/displayShow.mako [hunk 1]
```diff
@@ -41,7 +41,7 @@
                 % endif
             % endfor
             </select>
-        <div class="navShow"><img id="nextShow" src="${srRoot}/images/next.png" alt="&gt;&gt;" title="Next Show" /></div>
+        <div class="navShow"><img id="nextShow" src="${srRoot}/images/manualsearch.png" alt="&gt;&gt;" title="Next Show" /></div>
```
Comment (OmgImAlexis) [inline gui/slick/views/displayShow.mako]: Why is this using the manualSearch icon?

Comment (fernandog) [inline gui/slick/views/displayShow.mako]: Will fix it

**Comment by fernandog** [comment]
@medariox done!

**Comment by fernandog** [comment]
@p0psicles @OmgImAlexis need aproval

**Comment by OmgImAlexis** [comment]
Approved, looks fine to me.

**Pull request merged**